.wa-env

```diff
@@ -169,9 +170,12 @@ export TEST_LAW_FIRM_SHARE_CASE_D_PASSWORD=London03
 export TEST_JUDGE_X_USERNAME=ia.iacjudge.ccd@fake.hmcts.net
 export TEST_JUDGE_X_PASSWORD=London20
 
-export WA_TASK_DMNS_BPMNS_PATH="your path to DMNs repo"
-export WA_BPMNS_DMNS_PATH="your path to wa-standalone-task-bpmn repo"
-export IA_TASK_DMNS_BPMNS_PATH="your path to ia-task-configuration repo"
-
+export WA_TASK_DMNS_BPMNS_PATH="$(dirname "$(mdfind kind:folder 'ia-docker' | head -1)")/wa-task-configuration-template"
+export WA_BPMNS_DMNS_PATH="$(dirname "$(mdfind kind:folder 'ia-docker' | head -1)")/wa-standalone-task-bpmn"
+export IA_TASK_DMNS_BPMNS_PATH="$(dirname "$(mdfind kind:folder 'ia-docker' | head -1)")/ia-task-configuration"
 
+export TEST_LAW_FIRM_ORG_SUCCESS_USERNAME=ia-law-firm-success@fake.hmcts.net
+export TEST_LAW_FIRM_ORG_SUCCESS_PASSWORD=London08
```

Comment:
These projects are open-source. Although I'm aware that these passwords are meaningless as they are specific to the local environment, better not putting them in code available for everybody to read.

```suggestion
export TEST_LAW_FIRM_ORG_SUCCESS_PASSWORD=<your password here>
```

Reply:
Passwords are already in this file, and these specific ones are pulled from the .local_env file